aws/src/test/java/org/apache/iceberg/aws/s3/TestS3FileIOProperties.java

```diff
@@ -254,6 +255,11 @@ public void testS3FileIOProperties() {
         .containsEntry(
             S3FileIOProperties.REMOTE_SIGNING_ENABLED,
             String.valueOf(s3FileIOProperties.isRemoteSigningEnabled()));
+
+    Assertions.assertThat(map)
```

Comment:
There should also be a `Assertions.assertThat(s3FileIOProperties.writeStorageClass()).isNull();` check in `testS3FileIOPropertiesDefaultValues()`